socorro/schemas/socorro-data-1-0-0.schema.yaml

```diff
@@ -74,14 +90,13 @@ definitions:
       pattern_properties:
         type: object
         additionalProperties:
-          $ref": "#/definitions/node"
```

Comment:
This was a typo that prevented schema validation for anything in a `pattern_properties` field. :(